src/components/LocksCard.tsx

```diff
@@ -227,9 +220,9 @@ export const LocksCard = () => {
                 </PopoverTrigger>
                 <PopoverContent>
                   <p className="max-w-[15rem]">
-                    Funds unlockable after a certain time. These locks result
-                    either from a vote with conviction casted on a refrendum, or
-                    from undelegating.
+                    Funds unlock after a certain time. These locks result either
```

Comment:
I'd like to keep the same wording as the card, and if the card title is wrong, then let's change it. But the fact that users need to do something to get things unlocked makes me think "unlock" is not right. I agree unlockable is not great though..
```suggestion
                    Funds unlockable after a certain time. These locks result either
```